ACKategories/Classes/Color+Extra.swift

```diff
@@ -108,21 +108,21 @@ public extension UIColor {
 
     /// Create 1 x 1 px image from color.
     public var image: UIImage {
-        return image(CGSizeMake(1, 1))
+        return image(CGSize(width: 1, height: 1))
     }
 
     /// Create image from color with defined size.
-    public func image(size: CGSize) -> UIImage {
-        let rect = CGRectMake(0, 0, size.width, size.height)
+    public func image(_ size: CGSize) -> UIImage {
```

Comment:
It's UIColor extension and it generates solid color image of defined size. Why would we have resize method on UIColor? It doesn't make sense 😄